tests/test_metadata.py

```diff
@@ -398,9 +356,10 @@ def test_explicit_formatdata(self):
             drm_scheme=DeliveryMechanism.ADOBE_DRM,
         )
 
-        metadata = Metadata(formats=[drm_format],
-                            data_source=edition.data_source)
-        metadata.apply(edition)
+        circulation_data = CirculationData(formats=[drm_format],
+                            data_source=edition.data_source, 
+                            primary_identifier=edition.primary_identifier)
+        circulation_data.apply(pool)
```

Comment:
this test (and others that primarily use CirculationData) should be moved to the TestCirculationData class